scripts/addon/build.py

```diff
@@ -177,37 +207,54 @@ def retrieve_strings_message(manifest, filename):
         "comments": message_json.get("subtitle_comment", "Subtitle for a message view"),
     }
 
-    return retrieve_strings_blocks(message_json["blocks"], filename, message_strings, f"message.{message_id}")
-
-
-def write_en_language(filename, strings):
-    ts = ET.Element("TS")
-    ts.set("version", "2.1")
-    ts.set("language", "en")
-
-    context = ET.SubElement(ts, "context")
-    ET.SubElement(context, "name")
+    return retrieve_strings_blocks(message_json["blocks"], filename, message_strings, f"message.{message_id}", None)
 
-    for key, value in strings.items():
-        message = ET.SubElement(context, "message")
-        message.set("id", key)
+def retrieve_shared_strings_message(message_json, filename):
+    message_strings = {}
 
-        location = ET.SubElement(message, "location")
-        location.set("filename", "addon.qml")
+    addons_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../addons")
```

Comment:
nit: I think it would be more correct to make compute this path relative to the addon's manifest (eg: the `filename`) parameter to this function, or make it an argument to the `build.py` script. I think it works for now because there is only one strings.yaml for addons right now.

Note that there isn't just one addons directory that gets built - we also have the addons for functional testing (located in `tests/functional/addons` which might also contain their own `strings.yaml`

Reply:
Changed it to be relative to the filename, good call. (Decided not to add an argument to build.py until we hit a scenario where we'd be using it right away.)